src/navigation/sim_translator/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>sim_translator</name>
+    <version>0.0.0</version>
+    <description>Translates FS Sim topics to Driverless topics</description>
+    <maintainer email="team@qutmotorsport.com">Alistair English, Alastair Bradford</maintainer>
+    <license>MIT</license>
+
```

Comment:
would add `<depend>` for `driverless_msgs` + `driverless_common` so we can build `--packages-up-to` and will include that